### PR TITLE
adds feature flag for sign in page

### PR DESCRIPTION
**Pull request by CaitHawk**
Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

Adds feature flag for upcoming sign-in modal/USiP content changes regarding the deprecation of MHV and DL Logon credential service providers.

## Related issue(s)

[Jira Ticket](https://jira.devops.va.gov/browse/VI-379)

## What areas of the site does it impact?

```platform/utilities/feature-toggle/featureFlagNames.json```

## Acceptance criteria

Add feature toggles to the frontend (vets-website) and backend (vets-api) so that the new design is editable

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Review by rmessina1010 — DISMISSED** [review]
LGTM

File: src/platform/utilities/feature-toggles/featureFlagNames.json [hunk 1]
```diff
@@ -230,6 +230,7 @@
   "showUpdatedFryDeaApp": "show_updated_fry_dea_app",
   "showUpdatedToeApp": "show_updated_toe_app",
   "signInServiceEnabled": "sign_in_service_enabled",
+  "signInModalv2": "sign_in_modal_v2",
```
Comment (asg5704) [inline src/platform/utilities/feature-toggles/featureFlagNames.json:233]: Can you change that to capital `V` please

**Review by asg5704 — CHANGES_REQUESTED** [review]
Can you change to a capital `V` please

**Review by asg5704 — APPROVED** [review]

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**